core/camel/src/main/java/org/opennms/core/camel/MinionDTO.java

```diff
@@ -0,0 +1,118 @@
+/*******************************************************************************
```

Comment:
@cgorantla This DTO serves a different purpose. It is a base class that implements a generic message pattern similar to a Camel exchange: it has a Map<String,String> of header values and then a <body> containing an arbitrary message. We're using it to normalize the message format for syslogs and traps. We'll probably look at combining this with the RPC/Sink DTOs in the future.